guide/creating_element/testing.md

```diff
@@ -159,13 +144,64 @@ end
 
 Once again, we are comparing the pipeline's output with our reference file to assert that everything works correctly.
 
+### Customizing pipeline module
+
+[`Membrane.Testing.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Testing.Pipeline.html) allows us to provide our custom implementations of [`Membrane.Pipeline`](https://hexdocs.pm/membrane_core/Membrane.Pipeline.html) callbacks that will be executed prior to `Membrane.Testing.Pipeline`'s implementations. Let's create Pipeline that will answer to requests with response we passed in initialization options. To do this, we must define our custom module.
+
+```elixir
+defmodule Example.Pipeline do
+  use Membrane.Pipeline
+
+  def handle_init(options) do
+    state = %{response: options.response}
```

Comment:
Does Testing.Pipeline put something in options? If it does, I would mention that in the guide (warning that you should expect the unexpected in opts), otherwise, I would simply use options as the state.

Reply:
I am not 100% sure. But from what I understand Pipeline puts custom_args field of `%Testing.Pipeline.Options{}` here. Will check though